week-1/errors/4.js

```diff
@@ -1,2 +1,7 @@
-const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+const TwelveHourClockTime = "20:53";
+const twentyfourhourClockTime = "08:53";
+
+console.log(TwelveHourClockTime)
+console.log(twentyfourhourClockTime)
+
+//at first it was writtem as 12hourClockTime and 24HourClockTime, this was giving an error because I had a number int he variable, so I chnaged it to letters and it works 
```

Comment:
Well done on correcting the code. 
The reason is a little off - numbers are allowed in variable names, however, they can't be at the start of the variable name. For example, '1variable' is invalid but 'variable1' is valid.

Reply:
Thank you, i didn't know that.